### PR TITLE
docs: include  Helm charts and Docker to documents page

### LINKED ISSUE #341: Docs: Apisix Helm charts and Docker are not included in Documents
**Actual Behaviour**
Apisix Helm charts and Docker are not included in [Documents](https://apisix.apache.org/docs/)
 

**Expected Behaviour**
Docs contains all the documents we have, including Apisix Helm charts and Docker.

**Steps to reproduce it**
add a new field like docs on the config file including Apisix Helm charts and Docker
 
**Screenshots of the issue**
Screenshot (263)

 
**Would you like to work on the issue?**
yes I would like to work on the issue

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #341

Changes:

add a new field  **docs** on the config file including Apisix Helm charts and Docker

Screenshots of the change:

Screenshot (264)

### CODE REVIEW

**Comment by Yiyiyimu** [comment]
I guess we need different icons for different kinds of doc? @juzhiyuan

**Comment by juzhiyuan** [comment]
It would be better to update `shape` field

**Comment by KishaniKandasamy** [comment]
Hi, Is this okay? color and shape..

File: website/src/assets/icons/diamond.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="currentColor" class="bi bi-diamond-fill" viewBox="0 0 16 16">
+  <path fill-rule="evenodd" d="M6.95.435c.58-.58 1.52-.58 2.1 0l6.515 6.516c.58.58.58 1.519 0 2.098L9.05 15.565c-.58.58-1.519.58-2.098 0L.435 9.05a1.482 1.482 0 0 1 0-2.098L6.95.435z"/>
+</svg>
```
Comment (juzhiyuan) [inline website/src/assets/icons/diamond.svg]: image

Hi, we need to have a blank line at the end of files.

**Comment by KishaniKandasamy** [comment]
Hi, Is there anything that need to be changed in this PR?

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -14,6 +14,65 @@ module.exports = {
...
+        shape: "hexagon",
+        color: "#2563EB",
+        githubRepo: "apache/apisix-ingress-controller",
+        version: "0.5.0",
+        releaseDate: "2021-04-11",
+        firstDocPath: "/getting-started",
+      },
+      {
+        name: "APISIX™ Helm Charts",
+        nameInParamCase: "helm-charts",
```
Comment (Yiyiyimu) [inline website/docusaurus.config.js]:
```suggestion
        nameInParamCase: "helm-chart",
```

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -14,6 +14,65 @@ module.exports = {
...
+      {
+        name: "APISIX™ Docker",
+        nameInParamCase: "docker",
+        description: "Docker tooling for Apache APISIX.",
+        shape: "diamond",
+        color: "#FFD700",
+        githubRepo: "apache/apisix-helm-chart",
+        version: "1.0.0",
+        releaseDate: "2020-12-1",
+        firstDocPath: "/docker",
```
Comment (Yiyiyimu) [inline website/docusaurus.config.js]:
```suggestion
        firstDocPath: "/build",
```

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -14,6 +14,65 @@ module.exports = {
...
+        releaseDate: "2021-03-12",
+        firstDocPath: "/apisix",
+      },
+      {
+        name: "APISIX™ Docker",
+        nameInParamCase: "docker",
+        description: "Docker tooling for Apache APISIX.",
+        shape: "diamond",
+        color: "#FFD700",
+        githubRepo: "apache/apisix-helm-chart",
```
Comment (Yiyiyimu) [inline website/docusaurus.config.js]:
```suggestion
        githubRepo: "apache/apisix-docker",
```

File: website/docusaurus.config.js [hunk 5]
```diff
@@ -14,6 +14,65 @@ module.exports = {
     events: require("./static/data/events.json"),
     repoUrl: "https://github.com/apache/apisix",
     docsUrl: "",
+    docs: [
+      {
+        name: "APISIX™",
+        nameInParamCase: "apisix",
+        description: "A dynamic, real-time, high-performance API gateway.",
+        shape: "triangle",
+        color: "var(--ifm-color-primary)",
```
Comment (Yiyiyimu) [inline website/docusaurus.config.js]: why color here is not a certain value

**Review by Yiyiyimu — CHANGES_REQUESTED** [review]
Find several links could not been visited

File: website/docusaurus.config.js (hunk 5 above)
Comment (KishaniKandasamy) [reply website/docusaurus.config.js]: It is used in [downloads](https://github.com/apache/apisix-website/blob/master/website/docusaurus.config.js#L23)  and declared in [customTheme.css](https://github.com/apache/apisix-website/blob/master/website/src/css/customTheme.css#L5) so I left it as  it is.If the color must  be a certain value then I will change it:)

Comment (Yiyiyimu) [reply website/docusaurus.config.js]: No worries I'm just curious why is it not the same with others 🤣

**Review by Yiyiyimu — APPROVED** [review]
LGTM. Thanks for your contribution!

File: website/docusaurus.config.js (hunk 5 above)
Comment (KishaniKandasamy) [reply website/docusaurus.config.js]: okay:)

**Comment by KishaniKandasamy** [comment]
> Hi, Is there anything that need to be changed in this PR?

@juzhiyuan

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**